src/server/routes/sparql.js

```diff
@@ -68,6 +68,6 @@ const sparql = async (req, res, _next) => {
   res.end();
 };
 
-const routes = Router().get('/', sparql).post('/', sparql);
+const routes = Router().use('/', sparql).post('/', sparql);
```

Comment:
I think we can remove the `post(...)` here since the `get` was changed into a `use`